### DISCUSSION TITLE
Using a Fine-Grained PAT

### DISCUSSION THREAD

**Discussion by santaimpersonator** [squidfunk/mkdocs-material#5583] [Q&A]
Has anyone tried implementing the new, _fine-grained_ personal access token in their workflow?

- Currently, the _classic_ token works for us.
- However, we'd like to implement the fine-grained token to limit its privledges to read-only.

---

I noticed when setting up the token with our _bot_ account, it wasn't able to access the mkdocs-material-insiders repo under the **Repository access** section.

- I tried enabling the expected privledges and making them read/write.
  ![image](https://github.com/squidfunk/mkdocs-material/assets/36016723/9d8b298d-d03f-4ddf-81ba-2d5d798d433d)
  ![image](https://github.com/squidfunk/mkdocs-material/assets/36016723/fe89b174-ee5f-4fb2-b74c-7c6726738e84)
  However, this is the error message we get
  ```bash
  remote: Write access to repository not granted.
  fatal: unable to access 'https://github.com/squidfunk/mkdocs-material-insiders.git/': The requested URL returned error: 403
  ```

My guess is that the insiders repo would have to be under an organization and the bot account would have to be part of the organization to be able to select the appropriate **Resource owner** and specific **Repository access**.

**Comment by squidfunk** [comment]
We'll be moving the Insiders repository under an organization in the coming weeks/months. We can't just do that instantly, because there are hundreds of collaborators on that repository, so we need to think of a good migration path. When we started the Insiders program, it wasn't clear whether it would be a success, which is why we just started by going with a private user-level repo 😉

**Reply by santaimpersonator** [reply]
Oh, awesome... that is great news!
